### PR TITLE
add H264/uLaw Content Type definition; add MKV_PCM_xxx Codec ID defin…

### PR DESCRIPTION
**Pull request by dwaynehubbard**
Add video/mkvgen/Include.h definitions for the use of PCM Codec IDs and "video/h264,audio/mulaw" Content Type

These are dependencies for the addition of a new amazon-kinesis-video-producer-c sample project that uses H264 video with a single PCM u-law audio track.

This pull request goes with PIC issue 86.
https://github.com/awslabs/amazon-kinesis-video-streams-pic/issues/86

Addition of the following definitions to the appropriate video/mkvgen/Include.h sections:

#define MKV_H264_MULAW_MULTI_CONTENT_TYPE     ((PCHAR) "video/h264,audio/mulaw")

#define MKV_PCM_INT_LIT_CODEC_ID            ((PCHAR) "A_PCM/INT/LIT")
#define MKV_PCM_INT_BIG_CODEC_ID            ((PCHAR) "A_PCM/INT/BIG")
#define MKV_PCM_FLOAT_IEEE_CODEC_ID         ((PCHAR) "A_PCM/FLOAT/IEEE")

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/mkvgen/include/com/amazonaws/kinesis/video/mkvgen/Include.h [hunk 1]
```diff
@@ -150,6 +150,7 @@ extern "C" {
 #define MKV_AVC_CONTENT_TYPE                ((PCHAR) "video/avc")
 #define MKV_HEVC_CONTENT_TYPE               ((PCHAR) "video/hevc")
 #define MKV_H264_AAC_MULTI_CONTENT_TYPE     ((PCHAR) "video/h264,audio/aac")
+#define MKV_H264_MULAW_MULTI_CONTENT_TYPE     ((PCHAR) "video/h264,audio/mulaw")
```
Comment (MushMal) [inline src/mkvgen/include/com/amazonaws/kinesis/video/mkvgen/Include.h:153]: tab alignment

**Review by MushMal — COMMENTED** [review]
Looks good. Do you mind adjusting the tab please?

File: src/mkvgen/include/com/amazonaws/kinesis/video/mkvgen/Include.h (hunk 1 above)
Comment (dwaynehubbard) [reply src/mkvgen/include/com/amazonaws/kinesis/video/mkvgen/Include.h:153]: @MushMal gladly, I haven't seen tab settings documented anywhere. Would you mind telling me the project's tab settings so I can update both branches?

Comment (MushMal) [reply src/mkvgen/include/com/amazonaws/kinesis/video/mkvgen/Include.h:153]: For some of the projects we started enforcing clang format rules (WebRtc SDK). We will be doing it here too at some stage. Currently, we use some defaults which have been more organically established but soon will be the same as defined in WebRtc project.

In this case, the only ask is to align ((PCHAR) "video/h264,audio/mulaw") to the above line

Comment (hassanctech) [reply src/mkvgen/include/com/amazonaws/kinesis/video/mkvgen/Include.h:153]: @dwaynehubbard - if we can get the alignment to match the rest we should be able to push this through quickly!

**Comment by MushMal** [comment]
@hassanctech could I ask you please to merge this request? Let me know if you would rather me merging it

**Review by hassanctech — APPROVED** [review]

**Pull request merged**